lib/senders/index.js

```diff
@@ -9,9 +9,8 @@
 // there's nothing left that imports mailer/config, it is safe to merge
 // legacy_index.js and this file into one.
 var createSenders = require('./legacy_index')
-var P = require('../promise')
 
-module.exports = function (log, config, error, db, translator, sender) {
+module.exports = function (log, config, error, bounces, translator, sender) {
```

Comment:
Peeking in the SMS scripts, they already passed `null` for the db arg, seems like just leaving it as is such that `bounces = null` will be just fine.